charts/rancher-gke-operator/1.0.200/templates/deployment.yaml

```diff
@@ -25,3 +25,17 @@ spec:
           value: {{ .Values.httpsProxy }}
         - name: NO_PROXY
           value: {{ .Values.noProxy }}
+{{- if .Values.additionalTrustedCAs }}
+        volumeMounts:
+        - mountPath: /etc/ssl/certs/ca-additional.pem
```

Comment:
Is this always going to be the same path? I cannot recall if you can change the underlying distro in GKE, and if you can, if GKE overrides default paths anyway.

Reply:
This chart is installed on the local cluster. This is the same path that Rancher uses: https://github.com/rancher/rancher/blob/master/chart/templates/deployment.yaml#L136-L145

Reply:
This mountPath is inside the gke-operator pod, so it could be any path as long as the operator looks for it. We can keep using same one as rancher for uniformity sake. 
How does the operator pod use it from there ?

Reply:
It's put in that directory because the libraries automatically look in that place for CAs. It is automatically picked up and the pod/gke-operator doesn't need to do anything else.